### DISCUSSION TITLE
Pump Model update for PDF

### DISCUSSION THREAD

**Discussion by chunkysteveo** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#750] [General]
Hi - I have a pump model update to include in the PDF:

Name: Paris 2021
Pump: S100102
Wires: 6
Model: MIAMI2021
Confirmed working: PCB V1, LLC:1 (blue) - **NOTE:** 510Ω resistors on all data lines between LLC and CIO/DSP.

**Comment by chunkysteveo** [comment]
Have some V2B PCBs on the way to tidy up the wiring and add some caps, but it's working for me with some bodged resistors inline with the connector wires(!!)

**Reply by KingEB07** [reply]
Hey @chunkysteveo , you might help me with some info...can you confirm that V2B is working on this model S100102? Cheers!

**Reply by chunkysteveo** [reply]
> Hey @chunkysteveo , you might help me with some info...can you confirm that V2B is working on this model S100102? Cheers!

Hey! I've still got them in the box waiting to be wired up, sorry. I don't see why they wouldn't, as they are just a neater PCB version of my bodge job on the V1 with resistors.

I can let you know once soldered up, but no ETA yet on when I'll open the pump back up?!

**Reply by chunkysteveo** [reply]
> Hey @chunkysteveo , you might help me with some info...can you confirm that V2B is working on this model S100102? Cheers!

Getting there - soldered up. Ignore the extra bits - this is my own custom board with audio switch and temp probe connector. 

![20240901_134954](https://github.com/user-attachments/assets/0610c68d-c6a5-43e6-ae8a-1f5dbf237037)

**Reply by KingEB07** [reply]
👍 thanks for the update. I'll be giving it a go myself next week so hopefully all goes well!

**Comment by visualapproach** [comment]
Thx, noted in the manual for the next upload
